### ISSUE TITLE
Allow to easily install the most recent version of each branch

### ISSUE THREAD

**Issue by javiereguiluz** [symfony/symfony-installer#31]
As pointed by @xabbuh, currently is not easy to install the latest stable version of any supported branch. If you want to install the latest 2.3 version, you have to go to symfony.com website to check which is the latest version or you have to install `2.3.0` version and then upgrade.

We could offer the possibility of not providing the patch version, and this would install the latest version of the given branch:

```bash
$ symfony new blog/        # installs 2.5.5
$ symfony new blog/ 2.5.1  # installs 2.5.1
$ symfony new blog/ 2.5    # installs 2.5.5

$ symfony new blog/ 2.3.8  # installs 2.3.8
$ symfony new blog/ 2.3    # installs 2.3.20
```

**Comment by stof** [comment]
@javiereguiluz this is done already thanks to #90, isn't it ?

**Comment by Pierstoval** [comment]
:+1: I also think it's solved

**Comment by javiereguiluz** [comment]
You are all right. Let's close this issue as fixed by #90.
